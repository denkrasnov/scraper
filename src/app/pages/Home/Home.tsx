import React from "react";

import Box from "src/atoms/Box";
import Divider from "src/atoms/Divider";
import Header from "src/app/components/Header";
import SearchBar from "src/app/components/SearchBar";
// import CompareCard from "src/app/components/CompareCard";
import SearchResult from "src/app/components/SearchResult";

const Home = () => {
  return (
    <>
      <Header />
      <Divider />
      <Box flexDirection="column" padding="0 15px">
        <Box
          alignItems="flex-start"
          grow={1}
          justifyContent="center"
          margin="20px 0"
          width="100%"
        >
          <SearchBar />
        </Box>
        {/* <CompareCard /> */}
        <SearchResult />
      </Box>
    </>
  );
};

export default Home;
