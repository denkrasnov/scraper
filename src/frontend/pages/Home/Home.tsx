import React from "react";

import Header from "~app/components/Header";
import SearchResult from "~app/components/SearchResult";
import Menu from "~app/components/Menu";
import Divider from "~app/atoms/Divider";
import Box from "~app/atoms/Box";

const Home = () => {
  return (
    <Box flexDirection="column" height="100%">
      <Header />
      <Divider />
      <Box
        background="BACKGROUND"
        flexDirection="column"
        flexGrow="gr1"
        marginBottom="s24"
      >
        <Box
          alignItems="flexStart"
          flexDirection="column"
          justifyContent="center"
          marginBottom="s32"
          marginTop="s20"
          width="100%"
        >
          <Menu />
        </Box>
        <SearchResult />
      </Box>
    </Box>
  );
};

export default Home;
