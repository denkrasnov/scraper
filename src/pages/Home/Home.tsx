import React from "react";

import Box from "src/atoms/Box";
import Divider from "src/atoms/Divider";
import Header from "src/components/Header";
import SearchBar from "src/components/SearchBar";
import SearchResult from "src/components/SearchResult";

const Home = () => {
  return (
    <>
      <Header />
      <Divider />
      <Box flexDirection="column" marginBottom="25px" padding="0 5%">
        <Box
          alignItems="flex-start"
          grow={1}
          justifyContent="center"
          margin="20px 0 30px"
          width="100%"
        >
          <SearchBar />
        </Box>
        <SearchResult />
      </Box>
    </>
  );
};

export default Home;
