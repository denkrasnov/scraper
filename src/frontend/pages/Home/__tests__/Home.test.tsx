import React from "react";
import { shallow } from "enzyme";

import Home from "..";

describe("Home", () => {
  it("should render", () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
