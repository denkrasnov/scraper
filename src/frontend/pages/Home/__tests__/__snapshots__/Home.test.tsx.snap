// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render 1`] = `
<Box
  flexDirection="column"
  height="100%"
>
  <Grid
    flexGrow="gr1"
    main={true}
  >
    <Header />
    <Box
      flexDirection="column"
      gridArea="body"
    >
      <Box
        background="BACKGROUND"
        display="block"
        flexDirection="column"
        flexGrow="gr1"
        marginBottom="s24"
      >
        <SearchResult />
      </Box>
    </Box>
  </Grid>
</Box>
`;
